bears/shell/BashateBear.py

```diff
@@ -0,0 +1,32 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.PipRequirement import PipRequirement
+from coalib.settings.Setting import typed_list
+
+
+@linter(executable='bashate',
+        output_format='regex',
+        output_regex=r'\[.*\] (?P<origin>E\d+): '
```

Comment:
I think you should do `\[.*?\]` (note the question mark), that makes the expression non-greedy and the next immediate `]` is matched.